src/util/queue/api.ts

```diff
@@ -194,9 +197,9 @@ async function editTicket(id: string, ownerID: string, queueID: string, status:
 /**
  * Deletes a ticket with the given ID.
  */
-async function deleteTicket(id: string, queueID: string): Promise<void> {
```

Comment:
To check whether or not a request to delete a ticket is valid on the backend I decided to pass the status of the ticket (to check if the ticket is claimed) and a boolean representing whether or not the deletion requestion is from a ta (which is valid) or from a user (which is not).